asreview/webapp/start_flask.py

```diff
@@ -253,6 +259,10 @@ def create_app(**kwargs):
     if config_file_path != "":
         app.config.from_file(Path(config_file_path).absolute(), load=json.load)
 
+    # Make sure we have front end origins
+    if not app.config.get("FRONT_END_ORIGINS", False):
+        app.config["FRONT_END_ORIGINS"] = FRONT_END_ORIGINS
```

Comment:
I'm still a bit confused about this. In production mode, the front end is served on the same port as the backend. How does this work then?